packages/react-button/src/components/MenuButton/renderMenuButton.tsx

```diff
@@ -18,7 +18,7 @@ export const renderMenuButton = (state: MenuButtonState) => {
       <slots.icon {...slotProps.icon} />
       {contentVisible && <slots.content {...slotProps.content} />}
       {!iconOnly && <slots.menuIcon {...slotProps.menuIcon} />}
-      <slots.menu {...slotProps.menu} />
+      {(persistMenu || expanded) && <slots.menu {...slotProps.menu} />}
```

Comment:
When not expanded, but persistMenu=true, is the menu going to be correctly hidden/shown depending on the expanded state?

Reply:
right; whenever it is `!expanded`, the menu slot needs visibility: hidden pointer-events: none sort of rule. It needs to be positioned correctly, but non-interactive.

Reply:
It will be correctly hidden because we are passing the `hidden` prop correctly to the `menu`:

https://github.com/microsoft/fluentui/blob/1118b1f4cdb44aa19c543ee1b57a14a42aeb0f6c/packages/react-button/src/components/MenuButton/useExpanded.tsx#L79